packages/server/email/MailManagerMailgun.ts

```diff
@@ -1,15 +1,22 @@
-import Mailgun from 'mailgun-js'
+import Mailgun from 'mailgun.js'
+import FormData from 'form-data'
 import sendToSentry from '../utils/sendToSentry'
 import MailManager, {MailManagerOptions} from './MailManager'
 
 export default class MailManagerMailgun extends MailManager {
-  mailgun = new Mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.MAILGUN_DOMAIN})
+  //mailgun = new Mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.MAILGUN_DOMAIN})
+  mailgun = new Mailgun(FormData);
+  mailgunClient = this.mailgun.client({
+    username: 'api',
+    key: process.env.MAILGUN_API_KEY || '',
+    public_key: process.env.MAILGUN_PUBLIC_KEY
```

Comment:
looks like we only need a public_key if we use `parse` or `validate` (https://github.com/mailgun/mailgun-js#setup-client)
we don't use either of those, so we're good!

we could either add in the public key to all our .env files in action-devops and all our dokku instances, or we could just leave this out. i'd opt for leaving it out to keep it simple

Reply:
@mattkrick made the change per your guidance, thank you sir!